ui/components/atoms/A024-Popper/index.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import createPropTypes from 'json-schema-prop-types'
+import { withStyles } from '@material-ui/core/styles'
+
+import Popper from '@material-ui/core/Popper'
+
+import IconButton from '../../atoms/A017-IconButton'
+
+import schema from './schema'
+import styles from './styles'
+
+const A024 = props => {
+  console.log('PROPS', props)
+  return (
+    <div>
+      <IconButton
+        color={props.color}
+        disabled={props.disabled}
+        handleClick={props.handleClick}
+        iconname={props.iconname}
+        ariaLabel={props.ariaLabel}
+      />
+      <Popper>
```

Comment:
your not passing any props to popper?

you absolutely have to pass it open somehow to open it:

https://material-ui.com/api/popover/

```
open: If true, the popover is visible.
```

Reply:
And for testing look at how i did this for the drawer: https://github.com/root-systems/cobuy-atomic/blob/master/ui/components/organisms/O001%20-%20Layout/stories.js

```
32:       open={boolean('Toggle Drawer', true)}
```